### PR TITLE
Auto-updating AMI + Ruby 2.5

### PR DESCRIPTION
**Pull request by aviars**

### CODE REVIEW

File: deployment-configuration.md [hunk 1]
```diff
@@ -6,20 +6,22 @@ This document decribes how to install the software on various platforms.
...
 
 After your instance is loaded, the application will be acceisable on port 80 (the standard port for HTTP).
 Use the following link to jumpstart your deployment:
 
 
-https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#LaunchInstanceWizard:ami=ami-d0a778ad
+https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#LaunchInstanceWizard:ami=ami-833d94fe
 
 
 A `t2.micro` sized instance should be sufficient  for sites expecting a low about of traffic.
```
Comment (arscan) [inline deployment-configuration.md]: `amount`

File: deployment-configuration.md [hunk 2]
```diff
@@ -6,20 +6,22 @@ This document decribes how to install the software on various platforms.
...
-An AWS account is required to use AMI version. If you wish to run it locally, you can use Docker,
-or a number of other configuration options described below.
 
+It is important to open port `80` for HTTP  and port `22` for SSH if you need to gain shell access to the server.  Instead of clicking the "Review and Lunch" button, click next button until you get to the Security Groups option. Ensure 80 is accesible from anywhere and 22 is avaiable from an IP range from whcih you will be connecting.  Please see the image entitled "security-groups.png" in the deployment-files directory for an example of thius setting.
```
Comment (arscan) [inline deployment-configuration.md]: "Lunch", "accesible", "whcih", "thius".

File: deployment-configuration.md [hunk 3]
```diff
@@ -6,20 +6,22 @@ This document decribes how to install the software on various platforms.
...
+
+After this step is done, lauch the instance.  Obtain your instance's IP or host name from the AWS console. Point a web browser to the instance using the IP address or host name.
```
Comment (arscan) [inline deployment-configuration.md]: lauch

File: deployment-configuration.md [hunk 4]
```diff
@@ -37,7 +39,20 @@ and Unicorn on Ubuntu 16.
 
 2. Setup the crucible_smart_app.
 
-Issue the following commands to setup the crucible smart app.
+To use all features, install Ruby 2.5. Enter the following commands in your terminal.
```
Comment (arscan) [inline deployment-configuration.md]: Maybe say something like 

TLS connection testing requires Ruby 2.5 or greater.  To check to see what version of ruby is installed, type in the following command:

```
ruby --version
```
If you are not running version Ruby 2.5, you can install it using Ruby Version Manager (rvm) by issuing the following commands.

File: deployment-configuration.md [hunk 5]
```diff
@@ -37,7 +39,20 @@ and Unicorn on Ubuntu 16.
...
+
+
+    sudo apt-get install libgdbm-dev libncurses5-dev automake libtool bison libffi-dev
+    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+    curl -sSL https://get.rvm.io | bash -s stable
+    source ~/.rvm/scripts/rvm
+    rvm install 2.5.0
```
Comment (arscan) [inline deployment-configuration.md]: Can we do ```rvm install 2.5``` instead, to just do the most recent patch on 2.5 (does rvm work like this?)

Comment (aviars) [reply deployment-configuration.md]: the `.0` is required.  the `2.5` failed for me.

**Comment by aviars** [comment]
Closing in favor of #123

**Pull request closed without merging**